### PR TITLE
Model training

### PR DESCRIPTION
**Pull request by JoeHarrison**
getText() retrieves all tweets from the tweets.csv file and removes all the URLs with the removeURLS function. These tweets are split up into a training set and a test set. The training set is then put through a pipeline that uses a countvectorizer, tf-idf Transformer and Stochastic Gradient Descent Classifier.  Parameters for the functions in the pipeline were obtained by performing a grid search.
The data is then fitted and then trained.

### CODE REVIEW

**Review by Denpeer — APPROVED** [review]
Approved after hotfix 882889d
